src/chrome/content/rules/typescriptlang.org.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="typescriptlang.org">
```

Comment:
Please also add `typescriptlang.org` which can be redirected to `www`.

Reply:
Just adding the target won't work here, since `typescriptlang.org` does not support https. It needs to be redirected to `https://www.typescriptlang.org/` explicitly.

Reply:
Please add `<rule from="^http://typescriptlang\.org/" to="https://www.typescriptlang.org/" />` before `<rule from="^http:" to="https:" />`.

Reply:
Please also add a short comment about the issue on `https://typescriptlang.org/`